talentsearch/package.json

```diff
@@ -75,20 +79,25 @@
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-testcafe": "^0.2.1",
+    "jest": "^27.4.5",
     "laravel-mix": "^6.0.39",
     "laravel-mix-polyfill": "^3.0.1",
     "postcss": "^8.4.5",
     "prettier": "^2.5.1",
     "react-refresh": "^0.11.0",
     "storybook-addon-intl": "^2.4.1",
+    "ts-jest": "^27.1.2",
     "ts-loader": "^9.2.6",
     "typescript": "^4.5.4"
   },
   "dependencies": {
     "@heroicons/react": "^1.0.5",
-    "@types/react": "^17.0.37",
+    "@testing-library/react": "^12.1.2",
+    "@types/node": "^17.0.2",
+    "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
     "@types/react-select": "^5.0.1",
+    "babel-plugin-formatjs": "^10.3.12",
```

Comment:
I'm not sure these should be in the dependencies section.  Shouldn't they be devDependencies?

Reply:
changed the test folders as suggested. Will look into these dependencies

Reply:
Nice, the dependencies look better now.